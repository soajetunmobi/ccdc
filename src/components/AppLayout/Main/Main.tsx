import React, { FC, PropsWithChildren, ReactNode } from 'react'

export const Main: FC = ({ children }: PropsWithChildren<ReactNode>) => {
  return <main className='content'>{children}</main>
}
