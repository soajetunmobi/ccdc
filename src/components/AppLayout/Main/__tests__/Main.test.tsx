import React from 'react'
import { Main } from '../Main'
import { render } from '@testing-library/react'

test('it renders Main correctly', () => {
  const { container } = render(<Main />)
  expect(container).toMatchSnapshot()
})
