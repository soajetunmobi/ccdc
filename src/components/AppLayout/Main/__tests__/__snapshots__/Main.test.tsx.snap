// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Main correctly 1`] = `
<div>
  <main
    class="content"
  />
</div>
`;
